src/components/pages/CompanyPage/index.jsx

```diff
@@ -0,0 +1,676 @@
+                              key={i}
+                              icon={milestone.icon}
+                              color={milestone.color}
+                              inverted={i % 2 ? true : false}
+                            >
+                              <h4 className="font-weight-bold">
+                                {milestone.name}
+                              </h4>
+                              <p className="text-muted mt-2 mb-0">
+                                <MDBIcon icon="clock" aria-hidden="true" />{" "}
+                                {milestone.date}
+                              </p>
+                            </MDBTimelineStep>
+                          );
+                        })}
+                      </MDBTimeline>
+                    </div>
+                  )}
+                  {this.state.activeTab === 1 && (
+                    <div>
+                      <MDBRow>
+                        <MDBCol md="6">
+                          <p className="lead">
+                            {data.company.employees} employees
+                          </p>
+                        </MDBCol>
+                        <MDBCol md="6">
+                          <input
+                            type="text"
+                            className="form-control"
+                            placeholder="Search for employees"
+                          />
+                        </MDBCol>
+                      </MDBRow>
+                      <div className="charts my-4">
+                        <HorizontalBar
+                          data={this.state.dataHorizontal}
+                          options={{
+                            responsive: true,
+                            scales: {
+                              xAxes: [
+                                {
+                                  stacked: true,
+                                  ticks: {
+                                    min: 0,
+                                    stepSize: 1,
+                                  },
+                                },
+                              ],
+                            },
+                          }}
+                          height="100"
+                        />
+                      </div>
+                      <MDBRow>
+                        {data.employees.map((employee, i) => {
+                          return (
+                            <MDBCol md="4" key={i}>
+                              <MDBCard>
+                                <MDBCardBody>
+                                  <p className="font-weight-bold mb-0 d-inline-block clickable blue-text">
+                                    {employee.full_name}
+                                  </p>
+                                  <p className="text-muted">
+                                    {employee.position}
+                                  </p>
+                                  {employee.joined && (
```

Comment:
Please fix indentation.
```suggestion
                                    {employee.joined && (
```